packages/react-examples/src/react/ThemeGenerator/ThemeGenerator.doc.tsx

```diff
@@ -95,15 +95,16 @@ export class ThemeGeneratorPage extends React.Component<{}, IThemeGeneratorPageS
       this._fabricSlotWidget(FabricSlots.white),
     ];
 
-    const stylingUrl = 'https://github.com/microsoft/fluentui/tree/master/packages/styling';
+    const stylingUrl = 'https://github.com/microsoft/fluentui/tree/master/packages/style-utilities';
 
     return (
       <div className="ms-themer">
         <div className="overview">
           <h2 id="Overview">Overview</h2>
           <p>
-            This tool helps you easily create all the shades and slots for a custom theme. The theme can be used by
-            Fabric React's styling package, see the{' '}
+            {/* eslint-disable-next-line @fluentui/max-len */}
+            This tool helps you easily create all the shades and slots for a custom theme. The theme can be used by the{' '}
+            <code>@fluentui/style-utilities</code> package; see the{' '}
```

Comment:
(nit) maybe just reflow the line instead of disabling the line length warning?

Reply:
Unfortunately this is one of the handful of cases where prettier and eslint conflict--prettier thinks the line is "prettiest"(??) this way despite being slightly over the max length, but eslint doesn't have that exception. There's not an easy way to change how prettier formats things, so usually in cases like this we disable the lint rule one-off.